Assets/Scripts/PlayerController.cs

```diff
@@ -0,0 +1,62 @@
+using System.Collections;
+using System.Collections.Generic;
+using System;
+using UnityEngine;
+
+public class PlayerController : MonoBehaviour
+{
+
+    public int speed;
+    public Transform PlayerPosition;
+    // Start is called before the first frame update
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
```

Comment:
```suggestion
    void FixedUpdate()
```
Replace by **FixedUpdate()** method.